components/brave_rewards/resources/tip/lib/interfaces.ts

```diff
@@ -91,9 +86,11 @@ export interface HostError {
   code?: number
 }
 
+export type Balance = import('gen/brave/vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger_types.mojom.m.js').Balance
```

Comment:
I know that we have to do the odd "type import expression" thing in `d.ts` files, but can we just regular import here?

```js
import * as mojom from '../../shared/lib/mojom'

export interface HostState {
   // ...
  balanceInfo?: mojom.Balance
  // ...
}
```

Reply:
Good catch!